src/main/java/org/opentripplanner/transit/service/TransitModel.java

```diff
@@ -323,20 +343,34 @@ public void addFeedInfo(FeedInfo info) {
    * may become necessary when we start making graphs with long distance train, boat, or air
    * services.
    */
-  public ZoneId getTimeZone() {
-    return timeZone;
+  public Collection<ZoneId> getAgencyTimeZones() {
+    CalendarService cs = this.getCalendarService();
```

Comment:
```suggestion
```

Reply:
Fixed, this was probably a result of the recent mege